r/rsparkling/tests/testthat.R

```diff
@@ -4,5 +4,11 @@ library(sparklyr)
 library(h2o)
 library(rsparkling)
 
-test_check("rsparkling")
+options(rsparkling.sparklingwater.version = Sys.getenv("SPARKLINGWATER_VERSION","1.6.7"),
+        rsparkling.spark.version =          Sys.getenv("SPARK_VERSION","1.6.2"))
```

Comment:
`rsparkling.spark.version` is the new option which was only used in tests. I've added this here so we have single place to control spark and sw version. Both are using env vars to make it easy to change in CI.